test/unit/lib/database.test.js

```diff
@@ -86,12 +86,7 @@ test('Mongo Database tests', async dbTest => {
       let result = await Database()
       assert.deepEquals(result.$initialConnection, expectedResult.$initialConnection)
       assert.pass('Db connection successful')
-      process.nextTick(() => {
```

Comment:
This was closing the database connection too early, and the last 3 tests were failing. I'm not sure what purpose it has as the node process will exit with 0 should the tests pass (someone please correct me if I'm wrong).

I tested with the older version of node as well, and it also failed. Either there's something wrong with my environment, or this test has been broken for a little while.

Reply:
tests are not working properly on that one. I have some more tests in my local fork, but still not at the coverage we are aiming at. do not mind the tests atm.

Reply:
Ok no problem. I'll revert my changes.